sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -384,239 +384,166 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
     outputMode = Append
   )
 
-  // Inner joins: Multiple stream-stream joins supported only in append mode
-  testBinaryOperationInStreamingPlan(
-    "single inner join in append mode",
-    _.join(_, joinType = Inner),
-    outputMode = Append,
-    streamStreamSupported = true)
-
-  testBinaryOperationInStreamingPlan(
-    "multiple inner joins in append mode",
-    (x: LogicalPlan, y: LogicalPlan) => {
-      x.join(y, joinType = Inner).join(streamRelation, joinType = Inner)
-    },
-    outputMode = Append,
-    streamStreamSupported = true)
-
-  testBinaryOperationInStreamingPlan(
-    "inner join in update mode",
-    _.join(_, joinType = Inner),
-    outputMode = Update,
-    streamStreamSupported = false,
-    expectedMsg = "is not supported in Update output mode")
-
-  // Full outer joins: only batch-batch is allowed
-  testBinaryOperationInStreamingPlan(
-    "full outer join",
-    _.join(_, joinType = FullOuter),
-    streamStreamSupported = false,
-    batchStreamSupported = false,
-    streamBatchSupported = false)
+  // stream-stream join
+  {
```

Comment:
Let's remove the bracket for now, as it produces unnecessary changes. If we think it's better to have it to separate properly, we can add it after verifying all changes are OK.

Reply:
Similar to existing unit tests in this file, i.e. `streaming aggregation`, `FlatMapGroupsWithState`, `deduplicate`, I add bracket to group `stream-stream join` here, I think this is part of refactoring, as we move all relevant tests together, and it looks better. But it's just a pretty minor thing after all, and I am fine to remove it if people think that way.

Reply:
Without this bracket you can reduce many line changes which are hurdles to review. That can be added later (after reviewing) as I mentioned in my previous comment. Makes sense?

Reply:
cool, I got your point now, updated accordingly, thanks.